index.html

```diff
@@ -1,249 +0,0 @@
-  <link href="https://fonts.googleapis.com/css?family=Raleway&display=swap" rel="stylesheet">
-  <link rel="stylesheet" href="main.css" />
-  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
-  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Nanum+Gothic&display=swap">
-</head>
-
-<!-- Removes underlined hyperlinks -->
-<style>
-  A {
-    text-decoration: none;
-  }
-</style>
-<body>
-  <div class="container">
-    <div id="home-top">
-      <header>
-        <a href="index.html"><img src=beer-header.png /></a>
-        <hr><hr> 
-      </header>
-    </div>
-    <div id="horiz">
-    </div>
-    <div id="img-area">
-      <div id="tagline">
-        <h2>TAKE A LOOK AT THIS AMAZING <b> HOME PAGE!</b></h2>
-      </div>
-      <br />
-      <br />
-      <div class="img-top">
-
-        <div id="img-lft">
-          <img
-            src="https://images.pexels.com/photos/1095124/pexels-photo-1095124.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=250&w=175">
-        </div>
-        <div id="img-ctr">
-          <img
-            src="https://images.pexels.com/photos/3009802/pexels-photo-3009802.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=250&w=175">
-        </div>
-        <div id="img-rgt">
-          <img
-            src="https://images.pexels.com/photos/995686/pexels-photo-995686.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=250&w=175">
-        </div>
-      </div>
-      <br />
-    
-      <div style="text-align: center; "> <a href="learn-more.html"><button class="learnmore-button">TELL ME MORE </button></a></div>
-  </div>
-  </div>
-<br />
-<br />
-  <section id="quotebox"> <b>"BEER IS GOOD" </b> - Ben & Lauren</section>
-
-  <div class="main-contents">
-    <section id="main-paragraph">
-      <div class="second-title">
-        <h1> I DON'T WANT TO SAY <b>IT'S THE BEER </b>...<br> BUT IT'S THE BEER.</h1>
-        <img
-          src="https://images.pexels.com/photos/681847/pexels-photo-681847.jpeg?auto=compress&cs=tinysrgb&dpr=2&h=200&w=600">
-      </div>
-      <div class="sub-content-1">
-        <h3 id="subtitle-content-1">TOTALLY...</h3>
-      </div>
-      <div class="item-content-1">
-        <p>Et poutine unicorn, ea photo booth shabby chic fingerstache vegan slow-carb lomo est.
-        Keffiyeh air plant coloring book neutra waistcoat non affogato celiac do cliche intelligentsia poke. Seitan
-        kombucha af lorem pitchfork craft beer organic fingerstache eiusmod. Gluten-free locavore succulents eiusmod
-        cardigan vaporware. Church-key bicycle rights esse ut. Selfies small batch green juice, tbh shoreditch do tofu
-        mustache PBR&B adaptogen skateboard proident sartorial.</p>
-      </div>
-      <br>
-      <a href="continue-reading.html"><button class="reading-button" name="CONTINUE READING">CONTINUE READING</button>
-      </a>
-      <br>
-      <br>
-      <br>
-      <hr>
-      <hr>
-      <br>
-      <br>

```

Comment:
You could do this spacing through css rather than inserting a bunch of `br` elements which would make your html look a little cleaner.